adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/CommandHandler.java

```diff
@@ -198,4 +203,39 @@ private void cancelTimer(final Long timerId) {
         vertx.cancelTimer(timerId);
         LOG.trace("Canceled Timer [timer-id: {}}", timerId);
     }
+
+    /**
+     * A class to facilitate storing timer id, tenantObject, command subscription and commandContext in a map.
+     * 
+     */
+    private class WaitingForAcknowledgementMapValue {
```

Comment:
How about calling this *PendingCommandRequest* instead?
The JavaDoc is not very helpful as it only spells out what you can see from the class' structure. Please describe what the purpose of the class is instead ...

Reply:
Done